earn/src/components/advanced/modal/BorrowModal.tsx

```diff
@@ -232,6 +244,7 @@ export default function BorrowModal(props: BorrowModalProps) {
   }
 
   // TODO: use GN
+  // TODO: if !shouldWithdrawToWallet, max borrows should just be from `maxBorrow`, not `maxBorrowAndWithdraw`
```

Comment:
Should we implement this in this PR?

Reply:
We should probably do it soon, but it's not absolutely necessary. The new one would be strictly greater than the current one, so it's not like it'll cause the txn to fail.

And to be honest I don't feel like doing that math right now.